src/rdkafka.h

```diff
@@ -2370,13 +2370,34 @@ size_t rd_kafka_queue_length (rd_kafka_queue_t *rkqu);
  *
  * librdkafka will maintain a copy of the \p payload.
  *
+ * @remark IO and callback event triggering are mutually exclusive.
```

Comment:
I think this should actually be "@remark FD and callback event..." since they are both IO events.

Reply:
Dropped.